api/v1/types.go

```diff
@@ -58,6 +58,9 @@ const (
 	// PowerFlex - placeholder for constant powerflex
 	PowerFlex DriverType = "powerflex"
 
+	// PowerFlexName - placeholder for constant powerflex
```

Comment:
Why are we adding a new driver type.. Cant we have just one ... At line 59, I see powerflex and at line 62 vxflexos.. Why dont we go with powerflex and remove the lines from 61-63

Reply:
we use vxflexos in some part of our driver , similar to powerscale where it uses isilon in some parts of driver 
i guess we still need to have it